client/src/components/ReviewsOverview.jsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const ReviewsOverview = (props) => (
+  <div id="reviewsOverviewComponent">
+    <div className="avgReviewContainer">
+      <img src="./images/star-rating.png" width="70" height="30"></img>
```

Comment:
I think inline css is a bad thing. Maybe put into css file.

Reply:
Great comment, planning to switch this image out later with an icon so will update then. Thanks for catching that!